hive/datadog_checks/hive/data/conf.yaml.example

```diff
@@ -0,0 +1,863 @@
+instances:
+
+    ## @param host - string - required
+    ## HiveServer2 or Hive Metastore JMX host to connect to.
+    #
+  - host: localhost
```

Comment:
Isn't there a jmx_url parameter? Not sure which one is better.

Reply:
I don't know which one is better. Jboss is actually the only one to use `jmx_url`, so I don't think we ever discussed about that.